src/NServiceBus.AcceptanceTests/Sagas/When_doing_request_response_between_sagas.cs

```diff
@@ -125,7 +125,15 @@ void SendReply()
             }
         }
 
-        public class InitiateRequestingSaga : ICommand { }
+        public class InitiateRequestingSaga : ICommand
+        {
+            public InitiateRequestingSaga()
+            {
+                Id = Guid.NewGuid();
+            }
+
+            public Guid Id { get; private set; }
```

Comment:
Not sure it's good to have a message property as a private setter. Couldn't this mess with deserialization and result in mismatched Guids?